datalad/interface/common_cfg.py

```diff
@@ -30,6 +30,17 @@
 
 dirs = AppDirs("datalad", "datalad.org")
 
+
+def get_default_ssh():
+    from datalad.utils import on_windows
+    from pathlib import Path
+
+    windows_openssh_path = r'C:\Windows\System32\OpenSSH\ssh.exe'
+    if on_windows and Path(windows_openssh_path).exists():
+        return windows_openssh_path
```

Comment:
This is related to the [git-for-windows issue #3746](https://github.com/git-for-windows/git/issues/3748). Once this issue is resolved. this function can be removed and `"ssh"` can be used as default value in the definition of the config key `datalad.ssh.executable`.